src/content/pages/programme/voting.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Voting
+subtitle: Your opinion for a great conference!
+---
+
+# Community Voting
+
+Welcome to EuroPython Community Voting! We want feedback about what YOU would
+like to see at EuroPython 2025. We more than 572 proposals covering many
+aspects of Python, and we are excited to be putting together an engaging,
+educational and entertaining programme.  Your feedback is an important
+ingredient in this refinement and curation process.
+
+
+<div style={{textAlign: "center"}}>
+
+Community Voting is open
+
+from Wednesday, 5 February 23:00:00 UTC
+
+until Monday, 17 February 23:59:59 UTC
+
+Thank you for voting!
```

Comment:
```suggestion
<ButtonLink href="https://voting.europython.eu">Cast your votes now!</ButtonLink>
```
I think the "thank you for voting" was for after voting ends :) Now might be better to put the big button up? I think it is OK to repeat the button.